odoo/__init__.py

```diff
@@ -80,6 +80,21 @@ def registry(database_name=None):
         database_name = threading.currentThread().dbname
     return modules.registry.Registry(database_name)
 
+# ----------------------------------------------------------
+# module babel hack
+# make sure ICU [world] territory is "001"
+# whereas "unitag" notation is "AA"
+# http://www.localeplanet.com/icu/ar-001/index.html
```

Comment:
In terms of references, I would also point out to the actual reference used by `babel` for locales: the official [Unicode CLDR](http://cldr.unicode.org/), which is automatically imported by babel during builds, in order to produce the contents of `babel.localedata`.